linebot/models/messages.py

```diff
@@ -189,13 +189,14 @@ class StickerMessage(Message):
     """
 
     def __init__(self, id=None, package_id=None, sticker_id=None,
-                 sticker_resource_type=None, **kwargs):
+                 sticker_resource_type=None, keywords=None, **kwargs):
         """__init__ method.
 
         :param str id: Message ID
         :param str package_id: Package ID
         :param str sticker_id: Sticker ID
         :param str sticker_resource_type: Sticker resource type
+        :param list keywords: List of up to 15 keywords describing the sticker
```

Comment:
[nits]
how about specifying the type of keywords as `list[str]` with `type` directive?

Reply:
Agree, I will modify it.